skins/universal/dark/dark.skn

```diff
@@ -1,4 +1,4 @@
-<skin version="1.2" author="VitoZz" name="Dark" path="C:/Users/vital/AppData/Roaming/Psi+/skins">
+<skin version="1.2" author="VitoZz" name="Dark">
```

Comment:
removing base path should break the skin. as usually untested?

Reply:
@Ri0n: No image in this skin, it can be removed.

Reply:
have you tested this?